fe/src/main/java/org/apache/doris/load/loadv2/JobState.java

```diff
@@ -18,8 +18,9 @@
 package org.apache.doris.load.loadv2;
 
 public enum JobState {
-    PENDING,
-    LOADING,
-    FINISHED,
-    CANCELLED
+    PENDING, // init state
+    LOADING, // job is running
+    COMMITTED, // transaction is committed but not visible
```

Comment:
Is this enum persisted? If it is, we can't change its order. If it isn't, better to add comment for this enum

Reply:
we persist state by name, not by order, so it is OK.
And most of enum are persist by name, which is a common usage.